src/paths.ts

```diff
@@ -1,15 +1,7 @@
-import { t } from '@lingui/macro';
 import { ParamHelper, ParamType } from 'src/utilities';
 
 export function formatPath(path: Paths, data = {}, params?: ParamType) {
-  // insights router has basename="/", "/beta/" or "/preview/", with hub under a nested "ansible/automation-hub" route - our urls are relative to that
-  let url = IS_INSIGHTS
-    ? UI_BASE_PATH.replace('/preview/', '/')
-        .replace('/beta/', '/')
-        .replace(/\/$/, '')
-    : '';
-  url += (path as string) + '/';
-  url = url.replaceAll('//', '/');
+  let url = (path as string) + '/';
```

Comment:
```suggestion
  let url = (path as string) + '/';
  url = url.replaceAll('//', '/');
```

better keep the `replaceAll`, the `/` route can still cause a double slash